frontend/src/components/Election/Admin/AdminHome.tsx

```diff
@@ -153,6 +153,28 @@ const AdminHome = ({ election, permissions, fetchElection }) => {
                                 </StyledButton>
                             </Grid>
                             <Divider style={{ width: '100%' }} />
+                            <Grid xs={8} sx={{ p: 1 }}>
```

Comment:
This is outside the current PR, but in the future we should rework this page to avoid duplicate button definitions

Reply:
Agreed, I'll open a new issue.